Assets/PatchKit Patcher/Scripts/Debug/DebugMenu.cs

```diff
@@ -18,17 +19,11 @@ public class DebugMenu : MonoBehaviour
         private string _popupMessage;
         private GraphicRaycaster _graphicRaycaster;
         private PlatformType _platformType;
+        private bool _isStart = true;
```

Comment:
Please rename to `_wasInitialized` and invert boolean values (as the new name points to an inverted state).

Reply:
done